src/application/common/logger.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as Logger from 'bunyan';
+
+const ILogger = Symbol("ILogger");
+
+interface ILogger {
+    addStream(stream: Logger.Stream): Promise<void>;
```

Comment:
thinking about the second point again, if you use a logger with the same id on the frontend and backend sides, and the backend changes the level how the frontend gets to know about it? it could be updated with notifications... or we just prohibit such usages